src/main/resources/templates/db_scripts/schema.json

```diff
@@ -428,6 +428,12 @@
           "tOps": "ADD",
           "caseSensitive": false,
           "removeAccents": false
+        },
+        {
+          "fieldName": "metadata.updatedDate",
```

Comment:
Hi @julianladisch, @adamdickmeiss what type of index has to be created for following CQL `metadata.updateDate >='2020-05-12'` (i.e. for `>=` operator)?

Reply:
A b-tree index ("index" section). This is correct.